SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -352,6 +352,8 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 	EAutomationControllerModuleState::Type TestState = AutomationController->GetTestState();
 	bTestRunnning = TestState == EAutomationControllerModuleState::Type::Running;
 
+	CallBack(true);
```

Comment:
reckon this will break anything? it was odd that the callback was only called with false when the local deployment failed

Reply:
i initially thought there might be other code paths that didn't expect it to be true but seems like there aren't any that also pass a callback